Formula/imagemagick.rb

```diff
@@ -43,20 +44,20 @@ class Imagemagick < Formula
   depends_on "libtiff" => :recommended
   depends_on "freetype" => :recommended
 
-  depends_on :x11 => :optional
+  depends_on "fftw" => :optional
   depends_on "fontconfig" => :optional
+  depends_on "ghostscript" => :optional
+  depends_on "liblqr" => :optional
+  depends_on "librsvg" => :optional
+  depends_on "libwmf" => :optional
   depends_on "little-cms" => :optional
   depends_on "little-cms2" => :optional
-  depends_on "libwmf" => :optional
-  depends_on "librsvg" => :optional
-  depends_on "liblqr" => :optional
   depends_on "openexr" => :optional
-  depends_on "ghostscript" => :optional
-  depends_on "webp" => :optional
-  depends_on "openjpeg" => :optional
-  depends_on "fftw" => :optional
+  depends_on "homebrew/versions/openjpeg21" if build.with? "openjpeg"
```

Comment:
??? This formula no longer exists. `openjpeg21` is now `openjpeg` in the core, which is why this was changed.